src/sqlalchemy_declarative_extensions/dialects/postgresql/query.py

```diff
@@ -195,13 +196,23 @@ def get_procedures_postgresql(connection: Connection) -> Sequence[BaseProcedure]
 
 def get_functions_postgresql(connection: Connection) -> Sequence[BaseFunction]:
     functions = []
+    volatility_map = {
```

Comment:
i'd probably define this map on `FunctionVolatility` directly (can't recall if enums will let you do that or not), or as a method.

Reply:
That's a nice suggestion, I'll apply it!